src/platforms/web/util/class.js

```diff
@@ -37,7 +37,8 @@ export function renderClass (
   dynamicClass: any
 ): string {
   if (isDef(staticClass) || isDef(dynamicClass)) {
-    return concat(staticClass, stringifyClass(dynamicClass))
+    const concatenatedClassNames = concat(staticClass, stringifyClass(dynamicClass))
```

Comment:
What do you think about:

    const staticClasses = (staticClass || '').split(' ')
    const dynamicClasses = (dynamicClass || '').split(' ')
    return Array.from(new Set([...staticClasses, ...dynamicClasses])).join(' ')

instead of L39 - L85 ?

Reply:
This is what I wrote initially but then I couldn't get tests to pass in PhantomJS.

Reply:
Ok, I understand. Sorry I didn't check it before writing my comment.

 If you want you can probably use solution:

    return concat(staticClass, stringifyClass(dynamicClass))
      .split(' ')
      .filter((item, index, array) => index === array.indexOf(item))
      .join(' ')

Reply:
I have updated my changes. Thanks for the suggestion :)